linux_os/guide/system/auditing/auditd_configure_rules/audit_file_deletion_events/audit_rules_file_deletion_events.rule

```diff
@@ -54,3 +54,14 @@ ocil: |-
     {{{ ocil_audit_syscall(syscall="renameat") }}}
 
 {{{ ocil_clause_entry_audit_syscall() }}}
+
+warnings:
+    - general: |-
+        This rule aims to attest collection of events from multiple syscalls
```

Comment:
I'm not sure this sentence makes sense. I think you mean "This rule aims to test a collection"... or "attest to a collection of events"... But even then I'm not entirely sure it makes sense. Do you mind expanding on what you mean here?

Reply:
I've rephrased, It is actually shorter now, hope it is clearer.